docs/quick-start.md

```diff
@@ -61,7 +60,12 @@ You can also use custom configs to connect to an existing bcoin node
 
 ## Configuration
 
-Configuration shared between the `app` and `bcoin` containers.
+This section covers the shared configuration between the `app`
+and `bcoin` containers necessary for their communication.
+
+The configuration is managed via environment variables set in the `bcoin.env` file.
+Secrets are managed in a `secrets.env` file which is listed in the `.gitignore`.
+__NOTE: DO NOT CHECK THIS FILE IN TO VERSION CONTROL.__
```

Comment:
Should be clear that this is about the secrets file and not bcoin.env. e.g. "NOTE: The secrets.env file has sensitive information and should not be checked into version control" I think there was a sentence about that before the bold part but it looks like that's been taken out now.

Reply:
👍